.github/workflows/main.yml

```diff
@@ -22,5 +22,5 @@ jobs:
         run: docker-compose run --rm -e JEKYLL_ENV=production jekyll jekyll build
       - name: "Deploy"
         run: |
-          docker-compose run --rm -e AWS_ACCESS_KEY_ID=$AWS_ACCESS_KEY_ID -e AWS_SECRET_ACCESS_KEY=$AWS_SECRET_ACCESS_KEY aws s3 rm s3://help.keboola.com --recursive
-          docker-compose run --rm -e AWS_ACCESS_KEY_ID=$AWS_ACCESS_KEY_ID -e AWS_SECRET_ACCESS_KEY=$AWS_SECRET_ACCESS_KEY aws s3 cp _site s3://help.keboola.com --recursive --acl "public-read" --cache-control "max-age=600"
+          docker-compose run --rm -e AWS_ACCESS_KEY_ID=$AWS_ACCESS_KEY_ID -e AWS_SECRET_ACCESS_KEY=$AWS_SECRET_ACCESS_KEY aws s3 rm s3://help.keboola.com --recursive --region us-east-1
```

Comment:
Bucket is in `us-east-1` 
<img width="1414" alt="image" src="https://github.com/keboola/connection-docs/assets/903531/aec0ca03-4bdf-4eb0-a80d-39ce8ba62e06">